### PR TITLE
week3 Tasks

### PR DESCRIPTION
**Pull request by muhammedgamal12**

### CODE REVIEW

File: Tasks/src/Min_Max.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    public static void main(String[] args)
+    {
+        int [] numbers =new int[1000];
+        int min = numbers[0];
+        int max = numbers[0];
+
+        for ( int i = 0;i < numbers.length; i++)
+        {
+        numbers[i]=(int) (Math.random()*10000);
+        }
```
Comment (mohamedshaker9) [inline Tasks/src/Min_Max.java]: For better programming extract this code to new function called for example fillingArrayWithRandomNumbers

File: Tasks/src/Fibonacci.java
Comment (mohamedshaker9) [inline Tasks/src/Fibonacci.java:1]: Print the values

File: Tasks/src/Remove_Duplicates.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
+public class Remove_Duplicates {
```
Comment (mohamedshaker9) [inline Tasks/src/Remove_Duplicates.java]: The class name should be start with capital letter for each word. ex: RemoveDuplicates

File: Tasks/src/Common_Array.java [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        int [] new_array=common_members(arr1,arr2);
+
+        for (int i =0 ;i < new_array.length;i++)
+        {
+
+            System.out.print(new_array[i] + ",");
+        }
+
+    }
+    public  static int[] common_members(int []arr1 , int []arr2 )
```
Comment (mohamedshaker9) [inline Tasks/src/Common_Array.java]: method names should be written as camel case and start with a verb. ex, combineCommonValues

File: Tasks/src/Adding_TwoArrays.java
Comment (mohamedshaker9) [inline Tasks/src/Adding_TwoArrays.java:1]: sort the resulting array

File: Tasks/src/Difference_Elements.java
Comment (mohamedshaker9) [inline Tasks/src/Difference_Elements.java:1]: if this for task 12 you may not understood the task.
You Have only one array and you have to print the differnce between the minimum and maximum elements in the array.
for example:
intput: {1, 5, 7, 2, 10, 14, 12}
the result : 13

**Review by mohamedshaker9 — CHANGES_REQUESTED** [review]

File: Tasks/src/Fibonacci.java
Comment (muhammedgamal12) [reply Tasks/src/Fibonacci.java:1]: already printed , please can you explain more..

**Review by mohamedshaker9 — APPROVED** [review]

**Pull request merged**